### DISCUSSION TITLE
Activites

### DISCUSSION THREAD

**Discussion by chrissard2** [jlai/fitness-dashboard#1] [Q&A] [answered]
Per the fitbit message board you answered my question that we could be able to create activites.  That is good.

@Chrissard wrote:
Looked over your dashboard will you be able to edit activities, sleep or food?
@jl__ wrote:
There's an API for editing logged foods, but not activities/sleep/weight (create or delete only).

**Comment by gramapp** [comment]
How to delete an activity?

**Reply by jlai** [reply]
I just added a delete button to the top of the activity details dialog

**Reply by gramapp** [reply]
Unfortunately I don't see a delete button, tried following to no avail:
history>log activity> "+log activity"
dashboard>"+">"+Log activity"

Am I missing something?

![image](https://github.com/user-attachments/assets/7af0910b-c55d-47a6-8de6-71bc79504be7)

![image](https://github.com/user-attachments/assets/4f7057f1-2525-428e-a8a8-61efad0704f2)

Am I missing something?

**Comment by chrissard2** [comment]
You need to open up the activity, on the right about the download of TXC

![image](https://github.com/user-attachments/assets/da35d0a9-3119-4208-969e-23d24a2fe052)

**Reply by gramapp** [reply, marked as answer]
Oh yes. Thank you!!!!
